elog-errors.hpp

```diff
@@ -71,22 +71,22 @@ namespace Error
 
 namespace sdbusplus
 {
-namespace xyz
+namespace org
```

Comment:
Why do you need to change those name space in elog-errors.hpp?

Reply:
@alvintpwang I will fix it.